packages/web/src/common/store/pages/trending/lineups/trending/retrieveTrending.ts

```diff
@@ -68,6 +70,19 @@ export function* retrieveTrending({
     timeRange
   })
 
+  // DN may return hidden tracks in trending because of its cache
+  // i.e. when a track is in trending and the owner makes it hidden,
```

Comment:
this is good. it helps protect the creator at least and also prevent someone from ever SEEING a track that looks hidden on the trending page